### DISCUSSION TITLE
Apache reverse proxy for docker?

### DISCUSSION THREAD

**Discussion by gerroon** [searx/searx#1564] [Installation issue: Docker]
Hi

I am using the docker to run it, and it works well on the local network but I would like to proxy it over Apache. So I tried

`docker run -d --name searx -e IMAGE_PROXY=True -e BASE_URL=https://DOMAIN/searx -p 8080:8888 wonderfall/searx`

and in Apache config
```
ProxyPass /searx http://127.0.0.1:8080
ProxyPassReverse /searx http://127.0.0.1:8080
```

But this set up does not work, I can get the initial page with **broken layout**, and searching anything gives me

`The requested URL / was not found on this server.`

**Comment by dalf** [comment]
Can you try this : https://github.com/asciimoo/searx/issues/686#issuecomment-469916026 ?

**Comment by gerroon** [comment]
Hi

Thanks but I do not want to override my web root with searx, unless I misunderstood the solution.

**Comment by gerroon** [comment]
Btw I just tried it and it does not work. Also like I guessed it replaces the web root

**Comment by Pofilo** [comment]
A new docker repo can help you: https://github.com/searx/searx-docker.
Feel free to reopen the issue if needed.

**Comment by gerroon** [comment]
@Pofilo thanks but is this new docker container works well with apache (as in my original entry at the top)?

**Comment by Pofilo** [comment]
Oh sorry, it uses Caddy so not helping in your case.
I was closing a lot of issues related to docker, sorry for this one !
